include/RI/global/Blas-Fortran.h

```diff
@@ -28,15 +28,24 @@ extern "C"
 	float sdot_(const int*const n, const float*const X, const int*const incX, const float*const Y, const int*const incY);
 	double ddot_(const int*const n, const double*const X, const int*const incX, const double*const Y, const int*const incY);
 
+#ifdef __MKL
 	// d = Vx * Vy
 	//	reason for passing results as argument instead of returning it:
 	//	https://www.numbercrunch.de/blog/2014/07/lost-in-translation/
-	void cdotu_(std::complex<float>*const result, const int*const n, const std::complex<float>*const X, const int*const incX, const std::complex<float>*const Y, const int*const incY);
+	void cdotu_(std::complex<float>* const result, const int* const n, const std::complex<float>* const X, const int* const incX, const std::complex<float>* const Y, const int* const incY);

```

Comment:
@maki49 But MKL has its own C interface for BLAS, putting return value at the end of function parameters. I wonder if it will cause difference.

Reply:
> @maki49 But MKL has its own C interface for BLAS, putting return value at the end of function parameters. I wonder if it will cause difference. https://www.intel.com/content/www/us/en/docs/onemkl/developer-reference-c/2023-1/cblas-dotc.html

I think both the two ways are okay: one is `cdotc( result, n, x, 1, y, 1)`, the other is `cblas_cdotc( n, x, 1, y, 1, &result )`.
https://www.intel.com/content/www/us/en/docs/onemkl/developer-guide-macos/2023-0/call-blas-funcs-return-complex-values-in-c-code.html